ext/MODULE-SPEC.md

```
@@ -149,6 +149,39 @@ This only works at the prime `2`, resolving `RP_n^m`
+### (Stunted) Complex Projective Space
+
+This works at any prime, resolving `CP_n^m`. The cohomology is the subquotient
+of `F_p[u^±]` (where `|u| = 2`) with polynomial degrees between `min` and `max`.
+
+* `min`: The minimum polynomial degree. The corresponding topological degree is
+  `2 * min`. This can be negative.
+* `max`: The maximum polynomial degree. The corresponding topological degree is
+  `2 * max`. If unspecified, it is infinity.
+
+Example:
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)